user_service/main/models.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+class Profile(models.Model):
+    USER_ROLE_CHOICES = [
+        ('admin', 'Admin'),
+        ('normal user', 'Normal User'),
+    ]
+
+    user_id = models.OneToOneField(User, on_delete=models.CASCADE, primary_key=True) 
+    displayed_name = models.CharField(max_length=60)
```

Comment:
Is the `user_id` field the username? And the displayed name is different from that?

Reply:
The user_id refers to the id field in the user table.  

The user table has id and username, both have unique constraints.